sdcm/nemesis.py

```diff
@@ -2764,6 +2764,92 @@ def disrupt(self):
         self.call_random_disrupt_method()
 
 
+class CategoricalMonkey(Nemesis):
```

Comment:
please put this one into `COMPLEX_NEMESIS`
that means this one would be only called if defined specifically in `nemesis_class` configuration option